daemon/src/identify/protocol.rs

```diff
@@ -122,6 +125,9 @@ impl From<crate::Environment> for Environment {
         match environment {
             crate::Environment::Umbrel => Environment::Umbrel,
             crate::Environment::RaspiBlitz => Environment::RaspiBlitz,
+            crate::Environment::Citadel => Environment::Citadel,
```

Comment:
Proposal: change it to a String field which can be passed in. That way it is _dynamic_.

Reply:
I like this, as I never understood why we went with an enum in the first place. I will re-work this PR.

Reply:
turns out that this will break the Identify protocol. I don't think it's worth bumping the protocol just for this reason :/

Reply:
This should not break anything, because the taker will always receive an `Unknown` on the environment from the maker. It breaks it for the maker only and that is acceptable.

Reply:
perhaps I should have been more clear: 
- this changeset does not break anything,
- I tried to re-work this changeset into @bonomat 's suggestion and it will break, because suddenly we will pass a `String` instead of `Option<Enum>` in the identify protocol.

Reply:
> I tried to re-work this changeset into @bonomat 's suggestion and it will break, because suddenly we will pass a String instead of Option<Enum> in the identify protocol.

Are you sure this would have been a breaking change? 
We de/serialize it from a `String`, meaning, `Option<String>` should have the same effect as `Option<Enum>`.